### ISSUE TITLE
skipRequest works in GUI but not with postman-cli

### ISSUE THREAD

**Issue by nextrevision** [a85/POSTMan-Chrome-Extension#12506]
### Is there an existing issue for this?

- [X] I have searched the tracker for existing similar issues and I know that duplicates will be closed

### Describe the Issue

I have a collection testing the new `pm.execution.skipRequest()` method and I see it working within the GUI (request skipped) but not with postman-cli. I have installed the postman-cli fresh and when running the collection I get the following:

```
$ curl -o- "https://dl-cli.pstmn.io/install/linux64.sh" | sh
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  1298  100  1298    0     0   8689      0 --:--:-- --:--:-- --:--:--  8711
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 21.8M  100 21.8M    0     0  13.6M      0  0:00:01  0:00:01 --:--:-- 13.6M
The Postman CLI has been installed
$ postman --version
1.5.0
$ postman collection run [collectionId]
postman

mycollection

...

→ /SkipTest
  6⠄ TypeError in prerequest-script
  GET https://myendpoint.com/SkipTest [404 Not Found, 296B, 112ms]

...

 6.  TypeError                         Cannot read properties of undefined (reading 'skipRequest')
                                       at prerequest-script
                                       inside "/SkipTest"
```

The only thing in my Pre-request Script is

```
pm.execution.skipRequest()
```

### Steps To Reproduce

1. Create a new collection.
2. Add a test with a Pre-request Script with the content `pm.execution.skipRequest()`.
3. Execute the request via GUI (success).
4. Execute with postman-cli (failure).

### Operating System

macOS / Linux

### Postman Version

10.20.7

### Postman Platform

Postman App

### User Account Type

Signed In User

**Comment by vedkribhu** [comment]
@nextrevision thank you for reporting this, we are tracking this and it will be fixed in the next release of postman-cli.

**Comment by vedkribhu** [comment]
@nextrevision we have added the support for skipRequest in our latest postman-cli release (v1.6.0). Thanks again for reporting this.

**Comment by nextrevision** [comment]
Can confirm, seeing this fixed in v1.6.0.
